packages/frontend/src/pages/campaigns/filters/utils/index.ts

```diff
@@ -0,0 +1,38 @@
+import { Template } from "@carrot-kpi/sdk";
+
+export const CARROT_DEFAULT_TEMPLATE_ID = 1;
+export const CARROT_DEFAULT_ORACLE_TEMPLATE = 1;
+
+type CheckboxProp = {
+    id: number;
+    name: string;
+    checked: boolean;
+};
+
+export type CheckboxesObj = Record<number, CheckboxProp>;
```

Comment:
Should we rename this to just `Checkboxes`?

Reply:
I don't necessarily think it makes tooooo much sense to have this type when it's simply a rename, but I can see it having some value in refactoring scenarios (where you can only change the value of the type here and it changes everywhere).

Reply:
`CheckboxPropsMap` sounds decent, no?

Reply:
We ended up not needing it.

fixed: df180fc782eaa6d8a035701179412777fc890cba